evm_vs_ewasm_opcode_table.md

```diff
@@ -0,0 +1,125 @@
+0x3a: GASPRICE        |      getTxGasPrice
+0x3b: EXTCODESIZE     |      getExternalCodeSize
+0x3c: EXTCODECOPY     |      externalCodeCopy
+0x3d: RETURNDATASIZE  |      getReturnDataSize
+0x3e: RETURNDATACOPY  |      returnDataCopy
+0x40: BLOCKHASH       |      getBlockHash
+0x41: COINBASE        |      getBlockCoinbase
+0x42: TIMESTAMP       |      getBlockTimestamp
+0x43: NUMBER          |      getBlockNumber
+0x44: DIFFICULTY      |      getBlockDifficulty
+0x45: GASLIMIT        |      getBlockGasLimit
+0x54: SLOAD           |      storageLoad
+0x55: SSTORE          |      storageStore
+0x58: PC              |      -<sup>[†](#f1)</sup>
+0x5a: GAS             |      getGasLeft
+0xa0: LOG0            |      log(0,..)
+...                   |
+0xa4: LOG4            |      log(4,..)
+0xf0: CREATE          |      create
+0xf1: CALL            |      call
+0xf2: CALLCODE        |      callCode
+0xf3: RETURN          |      return
+0xf4: DELEGATECALL    |      callDelegate
+0xfa: STATICCALL      |      callStatic
+0xfd: REVERT          |      revert
+0xff: SELFDESTRUCT    |      selfDestruct
+
+
+†. <small id="f1">In EVM, the Program Counter (PC) opcode returns the currently executing bytecode position. When EVM is transpiled to ewasm with evm2wasm, this bytecode position is calculated by the transpiler and [inserted as a wasm constant](https://github.com/ewasm/evm2wasm/blob/80136977553c9b22e385f919fff808ef8a54be95/index.js#L247-L248). For ewasm code that is not transpiled from EVM, there is no interface method corresponding to the `PC` opcode.</small>
+
+
+## EVM opcodes vs WebAssembly instructions
+
+Whereas EVM is an untyped VM with a 256-bit word size, WebAssembly is a typed VM with 32-bit and 64-bit word sizes. The two wasm base types are integers and floats, which when combined with the two word sizes, create four types: i32, i64, f32, f64.
+
+Ethereum WebAssembly, or ewasm, is a subset of wasm. Only the integer types (i32 and i64) are supported, floating point types and floating point instructions are not supported. In total, 60 wasm instructions are supported in ewasm:
+* 10 control flow instructions
+* 11 basic instructions
+* 19 integer arithmetic instructions
+* 10 integer comparison instructions
+* 3 integer conversion instructions
+* 7 memory instructions (load, store, extend)
+
+ewasm excludes the wasm floating point instructions:
+* 14 floating point arithmetic instructions
+* 6 floating point comparison instructions
+* 7 floating point conversion instructions
+
+To reference wasm instructions, see the [wasm spec](https://github.com/WebAssembly/design/blob/master/Semantics.md) or [reference manual](https://github.com/sunfishcode/wasm-reference-manual/blob/master/WebAssembly.md#instructions).
+
+
+### EVM opcodes that reduce to wasm instructions
+
+EVM opcode (256-bit)  |    wasm instruction i32/i64 (32-bit or 64-bit)
+----------------------|----------------------
+0x00: STOP            |       end
```

Comment:
I think this is misleading. There is no specific `end` instruction so to speak, but rather the end of a function. It also will not stop execution if we are a nested call within wasm.

Actually, `STOP` is equivalent to `RETURN(0, 0)` on EVM. Based on that I'd just move it to the other table above.